src/wfs/src/test/java/org/geoserver/wfs/GetFeatureTest.java

```diff
@@ -398,6 +398,24 @@ public void testStrictComplianceBBoxValidator() throws Exception {
     }
 
     @Test
+    public void testStrictComplianceVersionNegotiation() throws Exception {
+        GeoServer geoServer = getGeoServer();
+        WFSInfo service = geoServer.getService(WFSInfo.class);
+        try {
+            service.setCiteCompliant(true);
+            geoServer.save(service);
+
+            Document doc = getAsDOM("wfs?service=WFS&request=GetCapabilities");
```

Comment:
Oh, the horror. This test fails if the `ows` endpoint is used. I am pretty sure all the CITE tests use the `ows` endpoint.